src/java/org/apache/cassandra/io/sstable/format/big/BigTableReader.java

```diff
@@ -311,11 +320,10 @@ protected BigTableRowIndexEntry getPosition(PartitionPosition key,
         return null;
     }
 
-
     @Override
-    public DecoratedKey keyAt(long dataPosition) throws IOException
+    public DecoratedKey keyAt(long position) throws IOException
```

Comment:
The other two `keyAt` versions must use the same file -- I think we need a `createKeyComponentReader` method to be used by `DecoratedKeyFetcher` and related.